### ISSUE TITLE
Auto status not working on DBus + Gnome + Wayland

### ISSUE THREAD

**Issue by Marcin100hi** [psi-im/psi#856]
Auto status not working when using DBUS + GNOME + WAYLAND

OS: Debian12, Linux 6.1.0-26-amd64, Wayland, Gnome 43.9

Psi Version: 1.5.2057

I compiled from source with FLAGS:
cmake -DPSI_PLUS=ON -DUSE_XSS=OFF -DUSE_DBUS=ON -DLIMIT_X11_USAGE=ON -DUSE_X11=OFF -DIRIS_BUNDLED_QCA=ON ..

Problem is in DBUS communication:
when call org.gnome.Mutter.IdleMonitor GetIdletime , return type is "qulonglong" , not "uint"

Probematic type is in line:
https://github.com/psi-im/psi/blob/9f94e0b66ea9fe213c109a88c23ca012bb0ca4f3/src/tools/idle/idle_x11.cpp#L151

reply is valid, when I change type from uint to qulonglong

```
if (interface.isValid()) {
            QDBusReply<qulonglong> reply = interface.call(method);
            // probably reply value for freedesktop and kde need to be converted to seconds
            if (reply.isValid())
		{
			//qDebug( "DBUS idle : %d", reply.value() );
                	return isNotGnome ? reply.value() / 1000 : uint(reply.value());
		}
	    else{
			qDebug( "DBUS Error Name : %s", reply.error().name().toUtf8().constData() );
			qDebug( "DBUS Error Msg  : %s", reply.error().message().toUtf8().constData() );
		}
        }
```

Best regards
Marcin

**Comment by Vitozz** [comment]
Yes, you are right. According to the Gnome documentation, it is uint64. To be honest, this code has never been tested in this environment.
But in KDE (plasma 6) with X11, the dbus method returns uint32. That is why I rewrote this mechanism. The response is now accepted as a QDBusMessage and the result is taken from the message argument as a QVariant with subsequent conversion to int.
The changes will be in the next commit.
